core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/definition/component/trace/componentDesign/HttpComponentHeaderDesign.java

```diff
@@ -0,0 +1,24 @@
+package io.metadew.iesi.metadata.definition.component.trace.componentDesign;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.UUID;
+
+@Data
+public class HttpComponentHeaderDesign {
+
+    @Builder
+    public HttpComponentHeaderDesign(UUID id, HttpComponentDesignTraceKey httpComponentDesignID, String name, String value) {
```

Comment:
HttpComponentDesignTraceKey should be ComponentDesignTraceKey,
id should be a HttpComponentHeaderDesignTraceKey